internal/common/resource.go

```diff
@@ -337,6 +356,14 @@ func mutate(f controllerutil.MutateFn, key client.ObjectKey, obj client.Object)
 	return nil
 }
 
+func isOwnedByVirtOperator(obj client.Object) bool {
+	if obj.GetLabels() == nil {
+		return false
+	}
+	managedValue, managedOk := obj.GetLabels()[kubevirtCorev1.ManagedByLabel]
```

Comment:
You can use constant `AppKubernetesManagedByLabel`, instead of the imported one `kubevirtCorev1.ManagedByLabel`.